packages/thirdweb/src/utils/ens/avatar.ts

```diff
@@ -1,20 +1,31 @@
-import { getCachedChain } from "../../chains/utils.js";
 import type { ThirdwebClient } from "../../client/client.js";
-import { getContract } from "../../contract/contract.js";
-import { isAddress } from "../address.js";
 import { getClientFetch } from "../fetch.js";
 import { resolveScheme } from "../ipfs.js";
+import { parseNftUri } from "../nft/parseNft.js";
 
-type AvatarOptions = {
+export type ParseAvatarOptions = {
   client: ThirdwebClient;
   uri: string;
 };
 
 /**
- * @internal
+ * Parses an ENS or similar avatar record. Supports NFT URIs, IPFS scheme, and HTTPS URIs.
+ * @param options - The options for parsing an ENS avatar record.
+ * @param options.client - The Thirdweb client.
+ * @param options.uri - The URI to parse.
+ * @returns A promise that resolves to the avatar URL, or null if the URI could not be parsed.
+ * @example
+ * ```ts
+ * import { parseAvatarRecord } from "thirdweb/utils/ens";
+ * const avatarUrl = await parseAvatarRecord({
+ *    client,
+ *    uri: "...",
```

Comment:
can you add examples to this? unsure when in what case i would use parseAvatarRecord vs parseNftUri

Reply:
Yeah I can, but 99% of the time you'd use parseAvatarRecord. I'm only exporting parseNftUri so social API can use it to parse the nft uri and return the ipfs hash (but not the ipfs url)